getting_started/placement.md

```diff
@@ -54,9 +54,10 @@ def main() = on[Client] {
     publicMessage.asLocal observe println
 
     // let the user send new messages via stdin
-    for (line <- scala.io.Source.stdin.getLines)
+    for (line <- scala.io.Source.stdin.getLines) {
       message.fire(line)
     }
+    }
```

Comment:
```suggestion
```
@JanisMittelstaedt I guess this one is not needed here. 

We already have one open and close brace for the `for`-loop and a closing one later for the method.